docs/mine/README.md

```diff
@@ -64,6 +66,12 @@ In Filecoin, miners earn two different types of rewards for their efforts: stora
 
 **Block rewards** are large sums that are given to the miner credited for a new block. Unlike storage fees, these rewards do not come from an associated client; rather, the network "prints" new FIL as both an inflationary measure and an incentive to miners advancing the chain. All active miners on the network have a chance at recieving a block reward, their chance at such being directly proportional to the amount of storage space currently being contributed to the network.
 
+### WinningPoSt
+WinningPoSt is the mechanism by which storage miners are rewarded for their contributions. In the Filecoin network, time is discretized into a series of epochs – the blockchain’s height corresponds to the number of elapsed epochs. At the beginning of each epoch, a small number of storage miners are elected to mine new blocks (Filecoin utilizes tipsets, which permit multiple blocks to be mined at the same height). Each elected miner who successfully creates a block is granted filecoin, as well as the opportunity to charge other nodes fees to include messages in the block.
```

Comment:
> is granted filecoin

Can you check what the styleguide says on capitalizing Filecoin when referring to the token, and PR an entry on that topic if one doesn't exist? My memory is that Filecoin is capitalized when it's referring to the token, but not 100% sure.